nodes/wot-discovery.js

```diff
@@ -147,6 +159,39 @@ module.exports = function (RED) {
 
             return addresses;
         }
+        function _getDiscoveryLinks(){
+            if(config.useCoap){
+                coapAddresses.forEach((address)=>{
+                    var reqOpt = url.parse(`coap://${address}/.well-known/core`);
+                    reqOpt.pathname = reqOpt.path;
+                    reqOpt.query =`rt=wot.thing`;
+                    reqOpt.multicast = true;
+                    var req = coap.request(reqOpt);
+                    req.on("response", _coreResponse);
+                    req.on("error", function (err) {
+                        node.log("client error");
+                        node.log(err);
+                    });
+                    req.end();
+                });
```

Comment:
Just some minor code styling. 

(Not entirely sure about the `const`s, though, but it should work.)

```suggestion
            if (config.useCoap) {
                coapAddresses.forEach(address => {
                    const reqOpt s= url.parse(`coap://${address}/.well-known/core`);
                    reqOpts.pathname = reqOpts.path;
                    reqOpts.query ="rt=wot.thing";
                    reqOpts.multicast = true;
                    const req = coap.request(reqOpts);
                    req.on("response", _coreResponse);
                    req.on("error", function (err) {
                        node.log("client error");
                        node.log(err);
                    });
                    req.end();
                });
```